### PR TITLE
End of Year: Implements design feedback for the listened numbers story

### PR DESCRIPTION
**Pull request by emilylaguna**
| 📘 Project: #374 | 🛫 Depends on: #561 |
|:---:|:---:|

This addresses the following items:
- Moves "kept" to a new line
- Updates the fonts
- Adjusts spacing between the covers
- I did not scale the covers up in size because the image assets we have became blurry.

### Design vs Implementation Comparison

<p align="center"></p>

### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
||||

## To test

1. Launch the app
2. Tap on Profile > End of Year card
3. Tap to the 5th screen
4. ✅ Verify it looks good and matches the designs
5. ✅ Repeat the steps of various screen sizes

## Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: podcasts/End of Year/Stories/Views/CommonStoryViews.swift [hunk 1]
```diff
@@ -194,3 +195,71 @@ private extension Double {
...
+    }
+
+    func body(content: Content) -> some View {
+        content
+            .rotationEffect(Self.rotationAngle, anchor: .center)
+            .scaleEffect(Self.scale, anchor: scaleAnchor)
+    }
+}
+
+struct PodcastPerspectiveRotator<Cover: View>: View {
```
Comment (emilylaguna) [inline podcasts/End of Year/Stories/Views/CommonStoryViews.swift:216]: This solves a problem when applying rotation effects which messes up the view's frame. This instead will calculate the proper height for it and reapply it after the rotation effect takes place.

File: podcasts/End of Year/Stories/ListenedNumbersStory.swift [hunk 2]
```diff
@@ -15,62 +15,52 @@ struct ListenedNumbersStory: ShareableStory {
...
+
+                    podcastCover(3)
+                        .frame(width: geometry.size.width * 0.35, height: geometry.size.width * 0.35)
+                        .padding(.leading, -(geometry.size.width / 4))
                 }
+                .applyPodcastCoverPerspective()
+                .padding(.top, geometry.size.height * 0.10)
+
+                let podcasts = L10n.eoyStoryListenedToPodcastText(listenedNumbers.numberOfPodcasts)
+                let episodes = L10n.eoyStoryListenedToPodcastText(listenedNumbers.numberOfEpisodes)
```
Comment (leandroalonso) [inline podcasts/End of Year/Stories/ListenedNumbersStory.swift]:
```suggestion
                let episodes = L10n.eoyStoryListenedToEpisodeText(listenedNumbers.numberOfEpisodes)
```

**Review by leandroalonso — CHANGES_REQUESTED** [review]
Tested and it looks good, but we're using the podcast text for the episodes.

File: podcasts/End of Year/Stories/ListenedNumbersStory.swift (hunk 2 above)
Comment (emilylaguna) [reply podcasts/End of Year/Stories/ListenedNumbersStory.swift]: Ah I made a change to fix that, but I don't see it anywhere in my commits. Fixed here: d7aa9e909552b97ec5f26e04d7b7a0deb9accc42

**Review by leandroalonso — APPROVED** [review]

**Pull request merged**